docs/ACCESSIBILITY.md

```diff
@@ -0,0 +1,139 @@
+```
+     _                         _ _     _ _ _ _
+    / \   ___ ___ ___  ___ ___(_) |__ (_) (_) |_ _   _
+   / _ \ / __/ __/ _ \/ __/ __| | '_ \| | | | __| | | |
+  / ___ \ (_| (_|  __/\__ \__ \ | |_) | | | | |_| |_| |
+ /_/   \_\___\___\___||___/___/_|_.__/|_|_|_|\__|\__, |
+                                                 |___/
+```
+
+<!--
+Standard by Glenn Chappell & Ian Chai 3/93 -- based on Frank's .sig
+Includes ISO Latin-1
+figlet release 2.1 -- 12 Aug 1994
+Modified for figlet 2.2 by John Cowan <cowan@ccil.org>
+  to add Latin-{2,3,4,5} support (Unicode U+0100-017F).
+Permission is hereby given to modify this font, as long as the
+modifier's name is placed on a comment line.
+
+Modified by Paul Burton  12/96 to include new parameter
+supported by FIGlet and FIGWin.  May also be slightly modified for better use
+of new full-width/kern/smush alternatives, but default output is NOT changed.
+
+Font modified May 20, 2012 by patorjk to add the 0xCA0 character
+-->
+
+# Focus management
+
+## Definitions
+
+### Focusable
+
+-  Interactive UI element that can be focused programmatically or via user gesture other than the <kbd>TAB</kbd> key, e.g. tap or mouse click
+-  It is optional to allow <kbd>TAB</kbd> key to focus on this element
+
+### Tabbable
+
+-  A [focusable](#focusable) which can be focused on by pressing <kbd>TAB</kbd> key
+
+## UX: A series of decisions
+
+> This is related to https://github.com/microsoft/BotFramework-WebChat/issues/3135.
+
+### User story
+
+The bot send a question with a set of predefined answers as UI buttons that will drive the conversation towards a particular goal (a.k.a. decision buttons).
+
+After the user make the decision by clicking on a button, the decision is being submitted. The user is not allowed to reselect another decision.
+
+The bot will then send another question with another set of answers.
+
+### Positive user experience
+
+We should disable buttons that are not part of the decision. Momentarily, the next set of answers will arrive. Since we must not change focus asynchronously outside of user gesture, the user is required to press <kbd>TAB</kbd> to get into the next set of decision buttons.
```

Comment:
The english on this was a little rough. Cleaned it up the best I could. Might want to make sure it still conveys everything correctly.

```suggestion
Once the user makes their selection, we should disable the decision buttons. Since the next question and set of possible decisions do not arrive immediately from the bot, we  can not change the focus asynchronously outside of user gestures. Consequently, the user is required to press <kbd>TAB</kbd> to move the focus to the next set of decision buttons.
```